src/utils/getResponse.ts

```diff
@@ -85,7 +85,9 @@ export const getResponse = async <
 
       const mockedResponse = await resolver(publicRequest, response, context)
 
-      if (!mockedResponse) {
+      // Skip a handler if there is no response or if it has been already used for
```

Comment:
@timdeschryver added a comment here similar to L34, let me know your thoughts, please.

Reply:
Looks good to me! 👍